tableauserverclient/models/site_item.py

```diff
@@ -694,15 +694,15 @@ def _parse_element(site_xml, ns):
         subscribe_others_enabled = string_to_bool(site_xml.get('subscribeOthersEnabled', ''))
         disable_subscriptions = string_to_bool(site_xml.get('disableSubscriptions', ''))
         revision_history_enabled = string_to_bool(site_xml.get('revisionHistoryEnabled', ''))
-        editing_flows_enabled = string_to_bool(site_xml.get('editingFlowsEnabled', ''))
-        scheduling_flows_enabled = string_to_bool(site_xml.get('schedulingFlowsEnabled', ''))
-        allow_subscription_attachments = string_to_bool(site_xml.get('allowSubscriptionAttachments', ''))
+        editing_flows_enabled = string_to_bool(site_xml.get('editingFlowsEnabled', 'true'))
```

Comment:
I think the default for the booleans should be an empty string (gets converted to false). For those who use older versions of Tableau Server, some of these newer settings are not returned by server. In that case, the default should be false rather than true.